### ISSUE TITLE
 the generated key points

### ISSUE THREAD

**Issue by cuichangrui** [Daniil-Osokin/lightweight-human-pose-estimation.pytorch#181]
Hello, I ran all the training steps you provided and got the accuracy, but I was wondering how to view the generated key points

**Comment by zhenzhongle** [comment]
https://github.com/Daniil-Osokin/lightweight-human-pose-estimation.pytorch/issues/119#issuecomment-736743042

**Comment by Daniil-Osokin** [comment]
Hope, it helped.
